{"name": "Невозможно авторизоваться с неверными учетными данными", "status": "passed", "start": 1740351834364, "stop": 1740351834972, "uuid": "aa12145f-3c93-41e0-94b0-0788af4f60ea", "historyId": "7cb8840ed5c768596e318cfc3f13414c", "testCaseId": "7cb8840ed5c768596e318cfc3f13414c", "fullName": "tests.test_courier.TestCourier#test_cannot_login_with_wrong_credentials", "labels": [{"name": "feature", "value": "Курьер"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_courier"}, {"name": "subSuite", "value": "TestCourier"}, {"name": "host", "value": "LAPTOP-QVE1U26H"}, {"name": "thread", "value": "2380-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_courier"}]}